client/app/account/user/credentials/user-credentials.html

```diff
@@ -0,0 +1,103 @@
+                        <th data-i18n-static="user_credentials_desc" scope="col"></th>
+                        <th data-i18n-static="user_credentials_creation_date" scope="col"></th>
+                        <th data-i18n-static="user_credentials_expiration_date" scope="col"></th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody data-ng-show="$ctrl.credentialsLoading">
+                    <tr>
+                        <td colspan="8"
+                            class="text-center">
+                            <oui-spinner data-size="s"></oui-spinner>
+                        </td>
+                    </tr>
+                </tbody>
+                <tbody data-ng-show="$ctrl.credentials.length === 0 && !$ctrl.credentialsLoading">
+                    <tr>
+                        <td colspan="8"
+                            class="text-center font-italic"
+                            data-i18n-static="user_credentials_table_empty">
+                        </td>
+                    </tr>
+                </tbody>
+                <tbody data-ng-show="$ctrl.credentials.length > 0 && !$ctrl.credentialsLoading">
+                    <tr data-ng-repeat-start="cred in $ctrl.credentials track by $index">
+                        <td>
+                            <button class="oui-button oui-button_link" title="{{ tr('user_credentials_show_permissions') }}"
+                                data-ng-if="!cred.isExpanded" data-ng-click="cred.isExpanded = true">
+                                <i class="oui-icon oui-icon-chevron-right"></i>
+                            </button>
+                            <button class="oui-button oui-button_link" title="{{ tr('user_credentials_hide_permissions') }}"
+                                data-ng-if="cred.isExpanded" data-ng-click="cred.isExpanded = false">
+                                <i class="oui-icon oui-icon-chevron-down"></i>
+                            </button>
+                        </td>
+                        <td data-ng-bind="cred.app.name"></td>
+                        <td data-ng-bind="cred.app.description"></td>
+                        <td data-ng-bind="cred.creation | date"></td>
+                        <td data-ng-bind="cred.expiration | date"></td>
+                        <td class="text-right">
+                            <button type="button"
+                                class="oui-button oui-button_secondary"
+                                data-ng-click="setAction('credentials/delete/user-credentials-delete', cred)"
+                                data-i18n-static="user_credentials_delete">
+                            </button>
+                        </td>
+                    </tr>
+                    <tr data-ng-show="cred.isExpanded" data-ng-repeat-end>
+                        <td colspan="3">
+                        <table class="table table-hover">
```

Comment:
For now, instead of nesting `<table>` you can provide two `<oui-action-menu-item>`.

The first one can open a modal window where we can access to the details of the application access and the second one is you can revoke.